Automattic/vip-go-mu-plugins
# Update ElasticPress to latest Automattic/ElasticPress#develop

## Description

Currently there's a bug with the VIP Search validate-counts command where the cache keys for post counts are the same for identical query args when iterating all subsites when using the `--network-wide` flag.

To account for this, we just need to append the cache key with the blog ID.

## Changelog Description

### Bug Fix: Append blog ID to VIP Search post count cache keys

There was a bug with the VIP Search validate-counts command where the cache keys for post counts are the same for identical query args when iterating all subsites when using the `--network-wide` flag. To account for this, we appended the blog ID to the cache keys.

https://github.com/Automattic/vip-go-mu-plugins/pull/this-pr

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. `wp vip-search health validation-counts` on a multisite.
2. Note how all DB counts for specific post types are the same for all subsites.
3. Apply PR.
4. `wp vip-search health validation-counts`
5. The DB counts should no longer be all the same for a specific post type on all subsites.
